backends/events/sdl/sdl-events.cpp

```diff
@@ -63,6 +70,31 @@ static uint32 convUTF8ToUTF32(const char *src) {
 
 	return utf32;
 }
+
+void SdlEventSource::loadGameControllerMappingFile() {
+	bool loaded = false;
+	if (ConfMan.hasKey("gcdbfile")) {
```

Comment:
`gcdbfile` is not a very explicit name. What about `controller_mapping_db`?

Also please document the new setting in the README, and mention `gamecontrollerdb.txt` will be loaded from `extrapath` otherwise.